<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monitorwidget</class>
 <widget class="QWidget" name="Monitorwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>569</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>启用指示灯功能</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QGroupBox" name="group_bit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>531</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>双态监视</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <widget class="QComboBox" name="comboBox_bitAddr">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>位地址</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>字地址</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>地址类型：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>字的位：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_bitIndex">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_bitAddr">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>70</y>
       <width>20</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resourse.qrc">
       <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_sameBit">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>与操作地址相同</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>71</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>监视地址：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_bitAddr">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>72</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_word">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>531</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>多态监视</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>地址类型：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_wordAddr">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>字地址</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>数据类型：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_dataType">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>16位正整数</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>32位正整数</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16位整数</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>32位整数</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>32位浮点数</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btn_wordAddr">
     <property name="geometry">
      <rect>
       <x>223</x>
       <y>150</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resourse.qrc">
       <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_wordAddr">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>LW0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_sameWord">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>与操作地址相同</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>监视地址：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>261</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>状态</string>
     </property>
     <widget class="QPushButton" name="btn_customValue">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>221</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>设定自定义状态值</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_statusCount">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>256</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>状态数：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_condition">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>按数值切换</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>按寄存器的位切换</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>自定义状态值切换</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>状态切换条件：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
